apps/andi/lib/andi_web/input_schemas/ingestion_metatadata_form_schema.ex

```diff
@@ -26,14 +26,39 @@ defmodule AndiWeb.InputSchemas.IngestionMetadataFormSchema do
     :targetDataset
   ]
 
+  def extract_from_ingestion_changeset(%Ecto.Changeset{ data: %Andi.InputSchemas.Ingestion{} } = ingestion_changeset) do
```

Comment:
Allows for extracting a child changeset from a parent changeset (ingestion -> metadata).

Allows for mapping fields (and errors) that don't line up identically for UI/Database separation of concerns.

This will allow all validation to happen at the API level (top level Andi parent changeset) instead of AndiWeb, then extract fields/errors down from there, mapping each as needed without the overhead of database migrations if you want to change a field name in an AndiWeb schema.

I would go as far to say, anything in AndiWeb.InputSchema should NOT have direct validation, so that all validation is made on API calls as well